cdap-app-fabric/src/main/java/io/cdap/cdap/internal/tethering/ArtifactCacheHttpHandlerInternal.java

```diff
@@ -69,12 +70,14 @@ public class ArtifactCacheHttpHandlerInternal extends AbstractHttpHandler {
   private final ArtifactCache cache;
   private final TetheringStore tetheringStore;
   private final RemoteAuthenticator remoteAuthenticator;
+  private final CConfiguration cConf;
 
   public ArtifactCacheHttpHandlerInternal(ArtifactCache cache, TetheringStore tetheringStore,
-      RemoteAuthenticator remoteAuthenticator) {
+      RemoteAuthenticator remoteAuthenticator, CConfiguration cConf) {
```

Comment:
Why do we need to pass cconf from here?

Reply:
Discussed offline, reverted the changes in this file.

I had changed it so that if someone wants to enable the internal router, the change gets applied uniformly everywhere.